### ISSUE TITLE
[nativescript-carousel] - NativescriptVue - indicatorOffset doesn't affect the indicator position

### ISSUE THREAD

**Issue by lruiz-empyrean** [nstudio/nativescript-plugins#99]
The app uses the nativescript-carousel to slide between components with a bottom indicator.

With the following configuration, regardless of the values set on indicatorOffset (even tried with negative values), the indicator is shown in the same position (bottom-center). 

Dependencies/DevDependencies
    "@nativescript/core": "^8.2.1",
    "@nativescript/theme": "^3.0.1",
    "@nstudio/nativescript-carousel": "^8.0.3",
    "nativescript-vue": "~2.9.0"
    "@nativescript/android": "^8.2.2",
    "@nativescript/ios": "^8.2.2",
    "@nativescript/types": "^8.0.0",
    "@nativescript/webpack": "^5.0.4",
    "@types/node": "17.0.10",
    "nativescript": "^8.1.5",
    "nativescript-vue-template-compiler": "^2.9.0",
    "typescript": "^4.2.3",
    "vue": "^2.6.12"

Template
`<Page>
    <GridLayout height="100%">
      <Carousel  ref="myCarousel" height="100%" width="100%" 
        showIndicator="true" indicatorColor="orange" indicatorColorUnselected="green" indicatorOffset="100,100"
        ios:finite="true"
        android:indicatorAnimation="slide" 
        :selectedPage="selectedIndex"
        @pageChanged="onSelectedIndexChanged">    
        <CarouselItem key="page-one">
         <PageOne />
        </CarouselItem>
        <CarouselItem key="page-two">
         <PageTwo />
        </CarouselItem>
         <CarouselItem key="page-three">
         <PageThree />
        </CarouselItem>
        <CarouselItem key="page-four">
         <PageFour />
        </CarouselItem>
        <CarouselItem key="page-five">
         <PageFive />
        </CarouselItem>
      </Carousel>     
    </GridLayout>
  </Page>`

**Comment by narender56** [comment]
Yup, Except indicatorColor nothing related to Indicators work. radius, padding, or offset doesn't work at all.

https://github.com/manijak/nativescript-carousel/issues/129#issuecomment-1647695818
